packages/fakta-beregning-redesign/package.json

```diff
@@ -24,6 +24,7 @@
     "storybook": "storybook dev --quiet -p 7002"
   },
   "dependencies": {
+    "@ft-frontend-saksbehandling/internal-components": "workspace:^",
```

Comment:
Denne bør ha same prefiks som dei andre pakkene: "@navikt/ft-". Og eg syns ikkje internal-components er noko godt funksjonelt navn. Det seier berre at dette er ei intern pakke, noko som er veldig tvetydig, Det er vanskeleg å sjå føre seg kva for kode som skal leggast her.